tools/topology/m4/muxdemux.m4

```diff
@@ -5,7 +5,23 @@ dnl Define macro for demux widget
 dnl Mux name)
 define(`N_MUXDEMUX', `MUXDEMUX'PIPELINE_ID`.'$1)
 
-dnl W_MUXDEMUX(name, mux/demux, format, periods_sink, periods_source, kcontrol_list)
+dnl CH_MAP_COEFF(coeff)
+define(`CH_MAP_COEFF',
+`	tuples."word.$1" {'
+`	SOF_TKN_CHANNEL_MAP_COEFF'		STR($2)
+`	}')
+
+dnl CH_MAP(name, id, ext_id, mask)
+define(`CH_MAP',
+`	tuples."word.$1" {'
```

Comment:
missing tokens "sof_chmap_tokens"?

Reply:
it is included in _mux_data_tuples_w which has the tokens "sof_chmap_tokens"... should I do it somehow differently?